test/runtime/specs/juttle-spec/expressions/function-call.spec.md

```diff
@@ -151,3 +151,106 @@ Produces an error when passed too many parameters (module)
 ### Errors
 
   * Error: function m.f expects 1 to 2 arguments but was called with 3
+
+Returns correct result when passed required parameters (without optional parameters)
+---------------------------------------------------------------------------------
+
+### Juttle
+
+    function f(a, b) {
```

Comment:
I’d just do `f(a)` in these tests, this would correspond to the existing tests a little bit more.

(We should really have more proper tests, passing 0, 1, and many parameters, but this is something for another day and another PR.)